azurerm/internal/services/avs/registration.go

```diff
@@ -0,0 +1,31 @@
+package avs
```

Comment:
presumably this package (and directory) wants to be `vmware` since that's what is it?